### PR TITLE
add kubernetes server/ingress/logging config to repo #1226

### PR DESCRIPTION
**Pull request by tbeerman**
add kubernetes server/ingress/logging config to repo #1226

### CODE REVIEW

File: etc/docker/kubernetes/README.rst
Comment (bari12) [inline etc/docker/kubernetes/README.rst]: a available -> are available.

Comment (bari12) [reply etc/docker/kubernetes/README.rst]: should be as well -> should work as well??

File: etc/docker/kubernetes/README.rst
Comment (bari12) [inline etc/docker/kubernetes/README.rst]: can have -> have

File: etc/docker/kubernetes/README.rst
Comment (bari12) [inline etc/docker/kubernetes/README.rst]: configuratio -> configuration
adeded -> added

File: etc/docker/kubernetes/README.rst
Comment (bari12) [inline etc/docker/kubernetes/README.rst]: to -> to the

File: etc/docker/kubernetes/README.rst
Comment (bari12) [inline etc/docker/kubernetes/README.rst]: that the -> that

**Review by bari12 — DISMISSED** [review]

File: etc/docker/kubernetes/README.rst
Comment (vingar) [inline etc/docker/kubernetes/README.rst:35]: rucio.cfg is missing in the repo. Would be good to add a reference one or the info about it

File: etc/docker/kubernetes/README.rst
Comment (vingar) [inline etc/docker/kubernetes/README.rst:25]: it's not clear for me. Would you have one example or latter in the documentation ?

File: etc/docker/kubernetes/README.rst
Comment (tbeerman) [reply etc/docker/kubernetes/README.rst:35]: the reference one is rucio.cfg.template in etc/

File: etc/docker/kubernetes/README.rst
Comment (tbeerman) [reply etc/docker/kubernetes/README.rst:25]: what's not clear about it? there is an example.

Comment (vingar) [reply etc/docker/kubernetes/README.rst:25]: sorry -- I'm learning kubernetes at the same time.   `kubectl get nodes` gives the list of nodes on the cluster.

**Comment by vingar** [comment]
@bari12 you're blocking this PR. Dismiss review ?

**Comment by bari12** [comment]
Oh I didn't realise. It's dismissed.

**Review by vingar — APPROVED** [review]

**Pull request merged**
